Applications/CLI/Tests.cmake

```diff
@@ -191,6 +191,32 @@ if(NOT OGS_USE_MPI)
         tes_zeolite_discharge_large_pcs_0_ts_28_t_1_000000.vtu tes_zeolite_discharge_large_pcs_0_ts_28_t_1.000000.vtu solid_density solid_density
     )
 
+   AddTest(
+        NAME 1D_HeatTransport_dirichlet
+        PATH Parabolic/T/1D_dirichlet
+        EXECUTABLE ogs
+        EXECUTABLE_ARGS line_60_heat.prj
+        WRAPPER time
+        TESTER vtkdiff
+        ABSTOL 1e-2 RELTOL 1e-2
```

Comment:
i have improved the tolerances from 1e-2 to 1e-4. The error between analytical solution and numerical solution is normally a little bit sloppy compared with the error between two numerical solutions